### PR TITLE
PAE-233 - Fix platform logo in course enrollment email

### PR DESCRIPTION
**Pull request by luismorenolopera**
## Description

Change the header logo in emails

#### Before
image

#### After
image

### CODE REVIEW

**Review by diegomillan — APPROVED** [review]

File: edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 1]
```diff
@@ -61,8 +61,8 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://image.freepik.com/vector-gratis/vector-onda-azul-transparente_1055-7084.jpg" width="70"
-                                    height="30" alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
+                                    src="https://mylearningx.pearson.com/static/pearson-pols-theme/images/logo.png" width="40"
```
Comment (Squirrel18) [inline edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html:64]: Can we get this value from site configuration? @luismorenolopera

File: edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 2]
```diff
@@ -61,8 +61,8 @@
...
+                                    height="40" alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
```
Comment (Squirrel18) [inline edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html:65]: Why height and not width? @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html:65]: both values ​​are assigned

File: edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html (hunk 1 above)
Comment (luismorenolopera) [reply edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html:64]: Being an email, I can't use branding api, or do you mean something else?

Comment (Squirrel18) [reply edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html:64]: Yes, the problem here is the async task...

File: edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html (hunk 2 above)
Comment (Squirrel18) [reply edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html:65]: Sorry, I didn't notice them.

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**